docs/system/process-metrics.md

```diff
@@ -20,32 +20,191 @@ metrics](/docs/runtime/README.md#metrics).
 <!-- toc -->
 
 - [Metric Instruments](#metric-instruments)
-  * [Process](#process)
-- [Attributes](#attributes)
+  * [Metric: `process.cpu.time`](#metric-processcputime)
+  * [Metric: `process.cpu.utilization`](#metric-processcpuutilization)
+  * [Metric: `process.memory.usage`](#metric-processmemoryusage)
+  * [Metric: `process.memory.virtual`](#metric-processmemoryvirtual)
+  * [Metric: `process.disk.io`](#metric-processdiskio)
+  * [Metric: `process.network.io`](#metric-processnetworkio)
+  * [Metric: `process.thread.count`](#metric-processthreadcount)
+  * [Metric: `process.open_file_descriptor.count`](#metric-processopen_file_descriptorcount)
+  * [Metric: `process.context_switches`](#metric-processcontext_switches)
+  * [Metric: `process.paging.faults`](#metric-processpagingfaults)
 
 <!-- tocstop -->
 
 ## Metric Instruments
 
-### Process
+### Metric: `process.cpu.time`
```

Comment:
We also specify here if the metric is required/recommended https://github.com/braydonk/semantic-conventions/blob/process_metrics/docs/system/system-metrics.md?plain=1#L68

Reply:
I think every metric in this document is going to be recommended. Should I add this to every section, or include something in the top?

Reply:
I'd say yes, to be consistent with the other files. I have opened an issue in the past, to make this part of the markdown template in the built-tool repo.

Reply:
Done in [914e018](https://github.com/open-telemetry/semantic-conventions/pull/330/commits/914e018823e3bcb85fc91eddb9b948ed4d13c2ea)